app/src/components/BaseTitle.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="px-4 md:px-12 pt-16 pb-8 border-b border-grey-100">
+    <div class="block md:flex gap-x-8">
+      <div class="flex flex-wrap content-center gap-x-8">
+        <div class="cursor-pointer uppercase font-medium">
+          {{ title }}
+          <span v-if="counter" class="ml-2 text-grey-400">({{ counter }})</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'BaseTitle',
+  props: {
+    title: { type: String, required: false, default: undefined },
+    counter: { type: Number, required: false, default: undefined },
```

Comment:
What is this prop for? Seems unused

Reply:
It's an optional counter which sits next to the title, we use this component multiple times on each page to separate sections - I've not seen where the counter is being used (yet) but it is part of @scco's designs - can you provide any more info here Richard?

Reply:
We can always remove it for now and add it back later as needed. Personally I'm not a huge fan of shipping unused code that may be used soon ™️

Reply:
It is gone! 

Yeah thats cool, I'm not either - its tricky knowing what to include and exclude when setting up components, it is probably the right move to just implement what we need though 👍